aten/src/ATen/native/Math.h

```diff
@@ -324,15 +329,20 @@ static inline double calc_digamma(double x) {
  * See note [3-Clause BSD License for the Cephes Math Library].
  */
 static inline float calc_digamma(float x) {
+  // See [CPP Standard Reference]
```

Comment:
"CPP Standard Reference" is a very generic name for a note. What about something more specific "Gamma Function C++ Standard Reference" or "C++ Standard Reference: Gamma Function"?